debug_toolbar/panels/settings.py

```diff
@@ -24,10 +24,16 @@ def title(self):
         )
 
     def generate_stats(self, request, response):
+        def catch_force_errors(force_function, value):
+            try:
+                return force_function(value)
+            except DjangoUnicodeDecodeError:
+                return "Debug toolbar was unable to parse value"
```

Comment:
```suggestion
                return "Django Debug Toolbar was unable to parse value"
```
nitpick: We use the full name in other messaging presented to users.